packages/components/src/components/page-overlay/page-overlay.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon';
```

Comment:
Can u import the icon.jsx file directly since it's a peer package component?
This will prevent the duplicated css since importing the index will cause the icon.scss to be imported again.